.gitignore

```diff
@@ -23,3 +23,7 @@ StortSpelProjekt/Game/imgui.ini
 StortSpelProjekt/Vendor/Resources/Models/Female/
 
 StortSpelProjekt/Vendor/Resources/Models/PlayerHuman/
+
+StortSpelProjekt/Vendor/Resources/Models/Demon/
```

Comment:
If you want to have one folder for all ignores I would like to ask that you remove these ignores from the gitignore.

Reply:
Yeah I'm the only one that has used it so it should be removed. 

Still keeping the rest though to avoid people forgetting to move over older models and accedentaly uploading stuff.